packs/equipment/howler-pistol.json

```diff
@@ -21,7 +21,7 @@
             "die": "d6"
         },
         "description": {
-            "value": "<p>Designed by a gunsmith as a personal challenge, a howler pistol strikes a miniature version of the sonic horn (see page 117) siege weapon's resonating orb with its hammer to generate beams of sonic energy. A howler pistol is a +1 striking dragon mouth pistol that deals sonic damage instead of piercing damage and doesn't have the concussive trait.</p>\n<p><strong>Activate—Resonant Shatter</strong> <span class=\"action-glyph\">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Requirements</strong> The howler pistol is loaded;</p>\n<hr />\n<p><strong>Effect</strong> You over-torque the howler pistol's hammer to strike the resonating orb harder than normal. The gun fires a concentrated burst of sonic energy that shatters a nearby stone or rock surface, creating a @Template[type:burst|distance:10] at a point of impact within the gun's first range increment. Creatures in the area take @Damage[5d6[piercing]] damage (@Check[type:reflex|dc:27|basic:true] save).</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include the powdered horn from a soniphak, processed into a resonating orb.</p>"
+            "value": "<p>Designed by a gunsmith as a personal challenge, a <em>howler pistol</em> strikes a miniature version of the <em>sonic horn</em> siege weapon's resonating orb with its hammer to generate beams of sonic energy. A <em>howler pistol</em> is a <em>+1 striking dragon mouth pistol</em> that deals sonic damage instead of piercing damage and doesn't have the concussive trait.</p><hr /><p><strong>Activate—Resonant Shatter</strong> <span class=\"action-glyph\">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Requirements</strong> The <em>howler pistol</em> is loaded</p>\n<p><strong>Effect</strong> You over-torque the <em>howler pistol</em>'s hammer to strike the resonating orb harder than normal. The gun fires a concentrated burst of sonic energy that shatters a nearby stone or rock surface, creating a @Template[type:burst|distance:10] at a point of impact within the gun's first range increment. Creatures in the area take @Damage[5d6[piercing]] damage (@Check[type:reflex|dc:27|basic:true] save).</p><hr /><p><strong>Craft Requirements</strong> The initial raw materials must include the powdered horn from a soniphak, processed into a resonating orb.</p>"
```

Comment:
Adding `area-damage` here, too
```suggestion
            "value": "<p>Designed by a gunsmith as a personal challenge, a <em>howler pistol</em> strikes a miniature version of the <em>sonic horn</em> siege weapon's resonating orb with its hammer to generate beams of sonic energy. A <em>howler pistol</em> is a <em>+1 striking dragon mouth pistol</em> that deals sonic damage instead of piercing damage and doesn't have the concussive trait.</p><hr /><p><strong>Activate—Resonant Shatter</strong> <span class=\"action-glyph\">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Requirements</strong> The <em>howler pistol</em> is loaded</p>\n<p><strong>Effect</strong> You over-torque the <em>howler pistol</em>'s hammer to strike the resonating orb harder than normal. The gun fires a concentrated burst of sonic energy that shatters a nearby stone or rock surface, creating a @Template[type:burst|distance:10] at a point of impact within the gun's first range increment. Creatures in the area take @Damage[5d6[piercing]|traits:area-damage] damage (@Check[type:reflex|dc:27|basic:true] save).</p><hr /><p><strong>Craft Requirements</strong> The initial raw materials must include the powdered horn from a soniphak, processed into a resonating orb.</p>"
```